plugins/woocommerce-admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -19,29 +18,28 @@ import { getSetting } from '@woocommerce/settings';
 import '../style.scss';
 import DismissModal from '../dismiss-modal';
 import SetupNotice, { setupErrorTypes } from '../setup-notice';
-import { getWcsAssets, acceptWcsTos } from '../wcs-api';
+import {
+	getWcsAssets,
+	acceptWcsTos,
+	getWcsLabelPurchaseConfigs,
+} from '../wcs-api';
 
 const wcAssetUrl = getSetting( 'wcAssetUrl', '' );
-const wcsPluginSlug = 'woocommerce-services';
+const wcsPluginSlug = 'woocommerce-shipping';
```

Comment:
Can we change the name of this to wcShippingPluginSlug instead, the current name is just too close to `wcstPluginSlug` which one can quickly misread, this is more of a code readability concern.

Reply:
Addressed in e8cdc699290fd621d2c5669b568e34752b02c37b